src/main/java/com/mraof/minestuck/computer/theme/ComputerTheme.java

```diff
@@ -0,0 +1,73 @@
+package com.mraof.minestuck.computer.theme;
+
+import com.google.gson.*;
+import com.mojang.logging.LogUtils;
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.JsonOps;
+import com.mraof.minestuck.Minestuck;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.GsonHelper;
+import net.minecraft.util.valueproviders.ConstantInt;
+import org.slf4j.Logger;
+
+import java.lang.reflect.Type;
+
+/**
+ * This class is used to define json files that will be used to determine what wallpaper and text color appears on in-game computers.
+ */
+public class ComputerTheme
```

Comment:
I'd like to suggest a rename to `ComputerThemeData` since this class strictly deals with the data of a theme.